### PR TITLE
feat

### PR DESCRIPTION
**Pull request by ckahub**
#CartPag
chrome-capture-2022-2-26 (2)
e

### CODE REVIEW

File: style/cart.css [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    height: 1rem;
+    line-height: 1rem;
+    justify-content:space-evenly;
+    align-items:center;
+    box-shadow: none;
+    text-align: center;
+}
+.nav2 ul{
+    list-style: none;
+    /* text-align: center; */
```
Comment (SanketDhabarde) [inline style/cart.css]: Remove this commented code

**Review by SanketDhabarde — APPROVED** [review]
- code structure looks good
- good use of variables
- good that you comment code for better understanding.

File: pages/cart.html [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+                        </div>
+                    </div>
+                </div>                                                                              
+            </div>
+        </div>
+    
+
+
+
+
```
Comment (AshwiniSetty) [inline pages/cart.html]: You can remove the extra spaces in the code

File: style/cart.css [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    background-color: #172b4df3;
+    width: 100%;
+    height: 15rem;
+    color: var(--secondary-color);
+    font-size: medium;
+    display:flex;
+    align-items:center;
+    justify-content: center;
+    text-align: center;
+}
```
Comment (AshwiniSetty) [inline style/cart.css]: I think by mistake you have specified footer CSS twice

File: style/cart.css [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+.btn.btn_Success{
+    background-color:var(--active-color);
+    color: var(--primary-color);
+}
+.btn.btn_Success:hover{
+    background-color:#1e9c1e;
+    color: white;
+}
+
```
Comment (AshwiniSetty) [inline style/cart.css:95]: Try storing the colors as variables and use it wherever needed

**Review by AshwiniSetty — APPROVED** [review]

File: style/cart.css (hunk 1 above)
Comment (ckahub) [reply style/cart.css]: removed

File: pages/cart.html (hunk 2 above)
Comment (ckahub) [reply pages/cart.html]: removed space

File: style/cart.css (hunk 3 above)
Comment (ckahub) [reply style/cart.css]: removed footer

File: style/cart.css (hunk 4 above)
Comment (ckahub) [reply style/cart.css:95]: one time thing

**Pull request merged**